src/content/documentation/publish/distribute-sideloading.md

```diff
@@ -15,13 +15,13 @@ date: 2019-11-06 04:05:42
 
 # Sideloading add-ons
 
-You may want to send a user your add-on’s XPI file by some means other than a web download, such as an email distribution of a beta version for user testing. In this case, there are two practical options for installing the add-on:
+You may want to send a user your add-on’s XPI file by some means other than a web download, such as an email distribution of a beta version for user testing. In this case, there are two options for installing the add-on:
 
-- Using **Install Add-on From File** in the Add-ons Manager.
+- Installing an extension from file in desktop Firefox or Firefox for Android.
 - Adding the file to one of the standard extension folders.
 
 ::: note
-No automatic updates will be performed for add-ons installed using these methods. You will have to deliver a new XPI file to your user for each update. However, automatic compatibility checks are still performed.
+No automatic updates are performed for add-ons installed using these methods. You have to deliver a new XPI file to your user for each update. However, automatic compatibility checks are performed.
```

Comment:
I don't think this is accurate. I tried walking through extensions source files on searchfox but I didn't see anything that suggested updates are handled different based on installation method.

@willdurand, can you confirm?

Reply:
@dotproto after reviewing the documentation, I believe this statement was actually intended to refer to extensions offered as a beta update release to a listed extension. I've updated the note.